css/properties/align-content.json

```
@@ -213,7 +213,8 @@
               "description": "<code>last baseline</code>",
               "support": {
                 "chrome": {
-                  "version_added": "108"
+                  "version_added": false,
+                  "impl_url": "https://crbug.com/341947689"
```

Was marked as duplicate. Want to use this instead?

```suggestion
                  "impl_url": "https://crbug.com/41344054"
```